src/EnergyPlus/Data/EnergyPlusData.hh

```diff
@@ -66,119 +68,95 @@ struct BaseGlobalStruct
 //    }
 //};
 
-struct DataGlobal : BaseGlobalStruct
-{
-    // Data
-    bool AnnualSimulation = false;
-
-    // MODULE VARIABLE DECLARATIONS:
-    std::string DayOfSimChr = "0";       // Counter for days (during the simulation) (character -- for reporting)
-
-    // MODULE PARAMETER DEFINITIONS
-    static constexpr int EndZoneSizingCalc = 4;
-
-    void clear_state() override {
-        AnnualSimulation = false;
-        DayOfSimChr = "0";
-    }
-};
-
-struct ExteriorEnergyUseData : BaseGlobalStruct
-{
-    struct ExteriorLightUsage
-    {
-        // Members
-        std::string Name;          // Descriptive name -- will show on reporting
-        int SchedPtr;              // Can be scheduled
-        Real64 DesignLevel;        // Consumption in Watts
-        Real64 Power;              // Power = DesignLevel * ScheduleValue
-        Real64 CurrentUse;         // Use for this time step
-        int ControlMode;           // Control mode Schedule Only or Astronomical Clock plus schedule
-        bool ManageDemand;         // Flag to indicate whether to use demand limiting
-        Real64 DemandLimit;        // Demand limit set by demand manager [W]
-        bool PowerActuatorOn;      // EMS flag
-        Real64 PowerActuatorValue; // EMS value
-        Real64 SumConsumption;     // sum of electric consumption [J] for reporting
-        Real64 SumTimeNotZeroCons; // sum of time of positive electric consumption [hr]
-
-                                   // Default Constructor
-        ExteriorLightUsage()
-            : SchedPtr(0), DesignLevel(0.0), Power(0.0), CurrentUse(0.0), ControlMode(1), ManageDemand(false), DemandLimit(0.0),
-            PowerActuatorOn(false), SumConsumption(0.0), SumTimeNotZeroCons(0.0)
-        {
+    struct DataGlobal : BaseGlobalStruct {
+        // Data
+        bool AnnualSimulation = false;
+
+        // MODULE VARIABLE DECLARATIONS:
+        std::string DayOfSimChr = "0";       // Counter for days (during the simulation) (character -- for reporting)
+
+        // MODULE PARAMETER DEFINITIONS
+        static constexpr int EndZoneSizingCalc = 4;
+
+        void clear_state() override {
+            AnnualSimulation = false;
+            DayOfSimChr = "0";
+        }
+    };
+
+    struct ExteriorEnergyUseData : BaseGlobalStruct {

```

Comment:
These structs should remain very lean, only variables.  No nested structures, no int const values, just the actual variables storing data.